android/rn-libs/react-native-randombytes/android/src/main/java/com/bitgo/randombytes/RandomBytesModule.java

```diff
@@ -0,0 +1,46 @@
+package com.bitgo.randombytes;
+
+import com.facebook.react.bridge.ReactMethod;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.Callback;
+
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+
+import java.security.SecureRandom;
+import java.util.Map;
+import java.util.HashMap;
+
+import android.util.Base64;
+
+public class RandomBytesModule extends ReactContextBaseJavaModule {
+  private static final String SEED_KEY = "seed";
```

Comment:
Is this the key we're gonna use? Maybe we can have a stronger one?

Reply:
The seed is default from the react-native-randombytes module